egs/grid/README.md

```diff
@@ -0,0 +1,45 @@
+Kaldi-based audio-visual speech recognition
+================================
+
+A baseline system for audio-visual speech recognition using the Kaldi speech recognition toolkit is provided.
+
+The scripts contain the early integration approach that is presented in:
+
+H. Meutzner, N. Ma, R. Nickel, C. Schymura, D. Kolossa, "Improving Audio-Visual Speech Recognition using Deep Neural Networks with Dynamic Stream Reliability Estimates", ICASSP, New Orleans, USA, March 2017.
+
+Future releases will also contain the late-integration approach using dynamic stream weights.
+
+
+Data description
+--------------------------
+
+The experiments are based on the audio data of the CHiME-2 challenge [1] and the video data of the GRID audio-visual speech corpus [2,3].
+
```

Comment:
Please describe that the data set is not exactly the same as for Chime as you remove one (?) speaker, i.e. the numbers are not directly comparable with the egs/chime2 recipe numbers.